src/Apps/Order2/Components/Order2PricingBreakdown.tsx

```diff
@@ -9,6 +9,9 @@ import { graphql, useFragment } from "react-relay"
 
 interface Order2PricingBreakdownProps {
   order: Order2PricingBreakdown_order$key
+  // TODO: Track from inside the component when details
+  // and checkout pages both call this
+  trackClickedImportFees?: () => void
```

Comment:
Because this is shared it's not easy to know where it gets clicked from (in terms of the context module). If we could use the URL or something instead that would make things easier.